### PR TITLE
Updated Monos chinos presence

### PR DESCRIPTION
**Pull request by ksprwhite**
**Name:**
Monos Chinos

**Changes:**
- Added new website url in metadata presence

**Screenshots:**

imagen
imagen

### CODE REVIEW

File: websites/M/Monos Chinos/dist/metadata.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
   "description": {
     "en": "Monos Chinos is a streaming website where you can watch anime with spanish subtitles."
   },
-  "url": "monoschinos.com",
+  "url": ["monoschinos.com", "monoschinos2.com"],
   "version": "1.0.5",
```
Comment (EncryptedDev) [inline websites/M/Monos Chinos/dist/metadata.json:11]: Bump the version

File: websites/M/Monos Chinos/presence.ts [hunk 2]
```diff
@@ -39,7 +39,7 @@ function getTimestamps(
   return [Math.floor(startTime / 1000), endTime];
 }
 
-presence.on("iFrameData", async (context) => {
+presence.on("iFrameData", async (context: any) => {
```
Comment (EncryptedDev) [inline websites/M/Monos Chinos/presence.ts:42]: No any type

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
Sup, check the review (:

File: websites/M/Monos Chinos/dist/metadata.json [hunk 3]
```diff
@@ -7,7 +7,7 @@
   "description": {
     "en": "Monos Chinos is a streaming website where you can watch anime with spanish subtitles."
   },
-  "url": "monoschinos.com",
+  "url": ["monoschinos.com", "monoschinos2.com"],
```
Comment (EncryptedDev) [inline websites/M/Monos Chinos/dist/metadata.json:10]: Why have two URLs if the other one is invalid?

**Review by EncryptedDev — CHANGES_REQUESTED** [review]
And this

File: websites/M/Monos Chinos/dist/metadata.json (hunk 3 above)
Comment (ksprwhite) [reply websites/M/Monos Chinos/dist/metadata.json]: ummm, this website uses the first url as redirection to second, I'm going to keep it with the second one

File: websites/M/Monos Chinos/presence.ts (hunk 2 above)
Comment (ksprwhite) [reply websites/M/Monos Chinos/presence.ts]: fixed

File: websites/M/Monos Chinos/dist/metadata.json (hunk 1 above)
Comment (ksprwhite) [reply websites/M/Monos Chinos/dist/metadata.json]: fixed

**Comment by EncryptedDev** [comment]
Show proof of it working too

**Comment by ksprwhite** [comment]
@EncryptedDev added screenshots

**Review by ririxidev — APPROVED** [review]

**Review by StrikerFRFX — APPROVED** [review]
lgtm

**Pull request merged**